magi-p3-post.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file should't be needed, right?

Reply:
Oh, vaadin-demo-helpers indirectly relies on vaadin-list-mixin^2.1.0 which causes a conflict. Nevermind then. We just need to make sure to remove this before final. Or use a magi post script (js) instead.